src/main.rs

```diff
@@ -291,12 +313,12 @@ struct CheckRelease {
     verbosity: clap_verbosity_flag::Verbosity<clap_verbosity_flag::InfoLevel>,
 
     /// Whether to print colors to the terminal:
-    /// always, alwaysansi (always use only ANSI color codes),
+    /// always, always-ansi (always use only ANSI color codes),
     /// auto (based on whether output is a tty), and never
     ///
     /// Default is auto (use colors if output is a TTY, otherwise don't use colors)
     #[arg(value_enum, long = "color")]
-    color_choice: Option<termcolor::ColorChoice>,
+    color_choice: Option<ColorChoice>,
```

Comment:
For some reason, this breaks the tests now because the `--color` arg is now incompatible with the `check-release` subcommand, which is a little weird because it's basically the same enum, and I didn't change anything in the `CheckRelease` struct besides that.  I can sort out the issue later, but if something jumps out at you for why this doesn't work suddenly, I'd love to know.

Reply:
I have very little experience with the `clap` crate, so I don't see anything obvious why this might be broken and I'm not sure I'd be much help on this.

Perhaps @pksunkara might have an idea though!

Reply:
To me, it looks like the test failures are related to removal of `CARGO_TERM_COLOR` logic unless I am mistaken.